frontend/packages/console-dynamic-plugin-sdk/src/api/flag.ts

```diff
@@ -0,0 +1,41 @@
+// eslint-disable-next-line @typescript-eslint/ban-ts-ignore
+// @ts-ignore: FIXME missing exports due to out-of-sync @types/react-redux version
+import { useSelector } from 'react-redux';
+import { RootState } from '@console/internal/redux';
```

Comment:
Plugin SDK should not depend on `@console/internal` package; this will complicate or even break SDK's TypeScript build & generated typings and put SDK's `package.json` dependencies into a discrepancy between what's declared vs. what's actually used.

We should pull the relevant code into the SDK package & update imports accordingly.

Any vendor libraries used in SDK APIs (such as `react-redux`) should also be explicitly mentioned in SDK's `package.json` dependencies.